src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala

```diff
@@ -19,8 +19,10 @@ package de.heikoseeberger.sbtheader
 import java.io.File
 import java.nio.charset.StandardCharsets.UTF_8
 import java.nio.file.Files
+
 import sbt._
-import sbt.Keys._
+import sbt.Keys.{ unmanagedSources, _ }
```

Comment:
Sorry, looks like IntelliJ was to ambitious when organizing imports :-) I'll revert that.